dataminer-overview/Support Services/Support_services_agreement.md

```diff
@@ -99,35 +97,12 @@ Skyline shall have no obligation of any kind to provide Support for Issues cause
 
 If Skyline determines that an Issue is a User-Generated Issue, Skyline will notify the User as soon as reasonably possible. If the User requests Support for the User-Generated Issue via a confirming email, Skyline shall have the right to invoice the User at Skyline's then current time and materials rates for any such Support provided by Skyline.
 
-## Pricing - Term
```

Comment:
Is the information in this section obsolete? If not, we will have to mention this elsewhere, because I don't think this info on pricing for the support services is mentioned under [Pricing](https://docs.dataminer.services/dataminer-overview/Pricing/Pricing.html) right now. The [FAQs](https://docs.dataminer.services/dataminer-overview/FAQ/FAQs_Support_Services.html) refer to this section here instead, so removing it causes an unresolved cross-reference.

Reply:
Thank you for bringing this up, Marieke. The information is not obsolete (yet). The Pricing page doesn't seem like the appropriate place to move this to; I'd rather move it directly to the FAQ page. We could add it as the first bullet point.

Reply:
Ok, I'll adjust this.